template.yaml

```diff
@@ -281,6 +281,14 @@ Resources:
       Handler: crowdaction
       Runtime: go1.x
       Events:
+        CreateCrowdaction:
+          Type: HttpApi
+          Properties:
+            Path: /cms/crowdactions
+            Method: post
+            ApiId: !Ref HttpApi
+            Auth:
+              Authorizer: "NONE"
```

Comment:
I am not sure if we should create an endpoint for creating a new crowdaction without any form of access control. Maybe we can add an AWS secret which is is passed as a parameter of the request?